pkgs/development/python-modules/expandvars/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "expandvars";
+  version = "0.9.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.4";
```

Comment:
Not a problem now but for future reference, v1.0.0 only supports Python >= 3.6 (https://github.com/sayanarijit/expandvars/commit/fe711567b82e3061ded33c94510eec31b20d9aac).

Reply:
Looks like v1.0.0 existed for a moment, and was then pulled again. The release is now named v0.11.0. I have updated the PR to address this, and bumped the requirement to Python >= 3.6.